firebase_to_xml/firebase_to_xml.py

```diff
@@ -0,0 +1,221 @@
+import traceback
+from google.oauth2 import service_account
+from google.auth.transport.requests import AuthorizedSession
+import json
+import os
+import sys
+import getopt
+import pathlib
+import pprint
+import metadata_xml.__main__ as metadata_xml
+from jinja2 import Environment, FileSystemLoader
+
+from dotenv import load_dotenv
+
+
+def main(argv):
+
+
+    load_dotenv()
+
+    # Load settings from .env file
+    key_file = os.getenv("FIREBASE_OAUTH2_KEY")
+    outputfolder = os.getenv("FIREBASE_OUTPUT_FOLDER")
+    # RA = os.getenv("FIREBASE_RA")
+    # user_list = os.getenv("FIREBASE_USER_LIST").split()
+
+    # load settings from command line
+    try:
+        opts, args = getopt.getopt(argv, "hk:o:", ["key=", "out="])
+    except getopt.GetoptError:
```

Comment:
could use `argparse`, but this works too

Reply:
changed in most recent push